src/Conversions/include/BipedalLocomotion/Conversions/ManifConversions.h

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file Conversions.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_MANIF_CONVERSIONS_H
+#define BIPEDAL_LOCOMOTION_MANIF_CONVERSIONS_H
+
+#include <iDynTree/Core/Transform.h>
+#include <iDynTree/Core/EigenHelpers.h>
+#include <manif/manif.h>
+
+
+namespace BipedalLocomotion
+{
+namespace Conversions
+{
+    /**
+     * @brief Convert rotation matrix and translation vector to manif SE3 object
+     *
+     * @param rotation reference to 3x3 Eigen matrix
+     * @param translation reference of 3x1 Eigen matrix
+     * @param pose reference to manif SE3 object
+     */
+    template <class Scalar>
+    void eigenRotTrans2ManifPose(const Eigen::Matrix<Scalar, 3, 3>& rotation,
+                                 const Eigen::Matrix<Scalar, 3, 1>& translation,
+                                 manif::SE3<Scalar>& pose)
```

Comment:
You may also define 
```cpp
manif::SE3<Scalar> eigenRotTrans2ManifPose(const Eigen::Matrix<Scalar, 3, 3>& rotation,
                                             const Eigen::Matrix<Scalar, 3, 1>& translation)

Reply:
Addressed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/138/commits/12d97ab9746193b054a09fd3be5e99ddd0b80836